doc/user/usage.md

```diff
@@ -241,7 +241,6 @@ ERRO[2016-11-07T19:53:44Z] vic-machine-linux failed: provide Common Name for ser
 
 The [`--cert-path`](#certificate-names-and---cert-path) option applies to all of the TLS configurations other than --no-tls.
 
-Using `--force` will allow a `create` operation to move past some certificate checks performed for existing certificates, however will do so by generating __new__ certificates, __overwriting__ the old.
```

Comment:
@gigawhitlocks I am fine with removing this from usage.md. I checked the user docs and the only thing that we say about using` vic-machine create --force` in relation to certs is that it skips the VC thumbprint check. So, I think that we're fine.